rust/ql/src/queries/telemetry/DatabaseQuality.qll

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
We already have a `rust/ql/src/queries/diagnostics` and `rust/ql/src/queries/summary` directory and this introduces `rust/ql/src/queries/telemetry` as well.  Are the new queries meaningfully different enough to justify a third location or should we condense back down to (any) two of them?

Reply:
I simply followed the same structure that we have for Java and C#.

Reply:
OK, I'll look into whether I can combine any of these directories after this PR is merged.  Don't really want a proliferation of folders unless they're meaningfully different.